game/CallsignExfil/Source/CallsignExfil/CallsignExfilCharacter.cpp

```
@@ -142,9 +142,31 @@ ACallsignNode* ACallsignExfilCharacter::GetCurrentNode_Implementation() const
+
+	// Detach from old node's occupancy.
+	if (CurrentNode && CurrentNode->Occupant.Get() == this)
+	{
+		CurrentNode->Occupant = nullptr;
+	}
+
 	CurrentNode = TargetNode;
-	// TODO Phase 1 impl: reposition the actor to the target node's transform
-	// (teleport for greybox, animated traversal later).
+
+	if (TargetNode)
```

Clang (14.0.6)

[warning] implicit conversion 'ACallsignNode *' -> bool

(readability-implicit-bool-conversion)